src/cmd/check.go

```diff
@@ -92,31 +92,72 @@ type CheckCreateType struct {
 	Spec map[string]interface{}
 }
 
-func (self *CheckCreateType) resolveAliases() {
+func (self *CheckCreateType) resolveCategoryAliases(client *opslevel.Client) (error) {
 	if item, ok := self.Spec["category"]; ok {
 		if value, ok := opslevel.Cache.TryGetCategory(item.(string)); ok {
 			delete(self.Spec, "category")
 			self.Spec["categoryId"] = value.Id.(interface{})
 		}
```

Comment:
We should add error handling here - if the user specified a category in the yaml but its not a valid value we should raise an error